demo/index.ts

```diff
@@ -66,12 +66,25 @@ const mockThirdPartyTracking = (html: string) =>
 
 const createCaptionPlugins = (schema: Schema) => exampleSetup({ schema });
 
+const additionalRoleOptions = [
+  { text: "inline (default)", value: "inline" },
```

Comment:
Should the value of this item be `undefinedDropdownValue` to more closely reflect Composer?